aspnetcore/blazor/hybrid/troubleshoot.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Troubleshoot ASP.NET Core Blazor Hybrid
+author: guardrex
+description: Learn how to troubleshoot issues in ASP.NET Core Blazor Hybrid with BlazorWebView logging.
+monikerRange: '>= aspnetcore-8.0'
+ms.author: riande
+ms.custom: "mvc"
+ms.date: 10/03/2023
+uid: blazor/hybrid/troubleshoot
+---
+# Troubleshoot ASP.NET Core Blazor Hybrid
+
+<!-- UPDATE 9.0 Activate after release and INCLUDE is updated
+
+[!INCLUDE[](~/includes/not-latest-version.md)]
+
+-->
+
+<xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView> has built-in logging that can help you diagnose problems in your Blazor Hybrid app.
+
+This article explains the two steps for this logging:
+
+1. Enable <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView> and related components to log diagnostic information.
+1. Configure a logger to write the log output to where to view logger output.
```

Comment:
I've had to parse this sentence a few times to get its meaning. Maybe it could be clearer?

Reply:
You're right ... I'll fix it.

Reply:
It was supposed to be ...

> Configure a logger to write the log output and where to view logger output.

... but even that's poor. I'll take another stab 🔪 at it.